src/panels/lovelace/views/default-view.ts

```diff
@@ -41,7 +41,7 @@ const getColumnIndex = (columnSizes: number[], size: number) => {
   return minIndex;
 };
 
-export class DefaultLayout extends LitElement implements LovelaceLayoutElement {
+export class DefaultView extends LitElement implements LovelaceViewElement {
```

Comment:
defaults can change, so let's give it a unique name.

Reply:
I guess Masonry is probably the best name for it